diff --git a/unity/Quackies.Unity/ProjectSettings/ProjectSettings.asset b/unity/Quackies.Unity/ProjectSettings/ProjectSettings.asset
index 786536c..eaf2907 100644
--- a/unity/Quackies.Unity/ProjectSettings/ProjectSettings.asset
+++ b/unity/Quackies.Unity/ProjectSettings/ProjectSettings.asset
@@ -150,8 +150,7 @@ PlayerSettings:
   visionOSBundleVersion: 1.0
   tvOSBundleVersion: 1.0
   bundleVersion: 1.0
-  preloadedAssets:
-  - {fileID: -944628639613478452, guid: 2bcd2660ca9b64942af0de543d8d7100, type: 3}
+  preloadedAssets: []
   metroInputSource: 0
   wsaTransparentSwapchain: 0
   xboxOneDisableKinectGpuReservation: 1
